docs/UserGuide.md

```diff
@@ -204,8 +209,12 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/client_TAG] [dt/
 * Existing values will be updated to the input values.
 * When editing tags, the tags specified using `t/` will be added to the contact (cumulatively).
 * Tags can also be removed using the delete tag `dt/` prefix, followed by the tag name.
+  * Any `dt/` arguments will **take priority** over any `t/` options, so `t/seller dt/seller` will **delete the `seller` tag**.
```

Comment:
I think you can mention here that it computes the overall change in the tags, so if you add a tag and delete it afterwards, it applies the adding of tags first and then deleting them.